packages/components/src/DatePicker/DatePicker.css

```diff
@@ -2,6 +2,10 @@
   display: inline-block;
 }
 
+.fullWidth {
```

Comment:
What is this fullwidth doing?

Reply:
(not a requested change per se, I'm just wondering what it does since it doesn't appear to have any utility in InputDate?)

Reply:
If we specify `inline` it makes it **not** full width which is how the `fullWidth` is being used internally. Check this:

![image](https://user-images.githubusercontent.com/88052089/143503207-158a9e97-07b3-4e84-bed5-64d95f5de73a.png)

and 
[here](https://github.com/GetJobber/atlantis/pull/783/files#diff-92e53204825907e717a1a30151cbb9929a8eef77e94837c2e7cd608ccc10f625R33)

Reply:
ooooh ok, gotcha